portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  </head>
+  <body>
+
+      <h1 class="jumbotron"> <center> Gallery </center> </h1>
+<nav>
+      <ul>
+   <li><a href="index.html">Home</a></li>
+  <li><a href="about.html">About</a></li>
+  <li><a href="gallery.html">Gallery</a></li>
+  <li><a href="poems.html"> Favorite Poems</a></li>
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
+      </ul>
+</nav>
+      
```

Comment:
If this is pasted in each file, is it possible to have this be in a separate file that you import?